docs/blockchain/why-us-aleph.md

```diff
@@ -0,0 +1,58 @@
+- Feature Name: (fill me in with a unique ident, `my_awesome_feature`)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: [pyrsia/pyrsia#0000](https://github.com/pyrsia/pyrsia/pull/0000)
+- Pyrsia Issue: [pyrsia/pyrsia#0000](https://github.com/pyrsia/pyrsia/issues/0000)
+
+# Summary
+
+Why we chose to use Aleph for our consensus engine
+
+# Motivation
+
+One of Pyrsia's main design decisions is to use a blockchain for its underlying datastore. The data structure(s) of a
+blockchain are plenty challenging but nothing mind bending. However, the real challenge to blockchain is consensus;
+how do you get a bunch of strangers to trust each other and agree, when some of them might be corrupt and you could
+have any number of faults in the system?
+
+# Our core requirements #
+- No currency. We wish to avoid legal problems
+- High TPS and TTF
```

Comment:
TPS does not matter and TTF should be small

```suggestion
- Short TTF
```

Reply:
Please provide the full form of an abbreviation at first use